src/styles/index.scss

```diff
@@ -2,2 +1,62 @@
+@import './variables';
+
+.page {
   margin: 0;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  height: 100vh;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 1px;
+  padding: $calendar-padding;
+  width: $calendar-width;
+
+  &__day {
+    width: $cell-size;
+    height: $cell-size;
+    box-sizing: border-box;
+    border: $cell-border;
+    background-color: $cell-bg-color;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    transition:
+      transform 0.5s,
+      background-color 0.5s;
+
+    &:hover {
+      cursor: pointer;
+      background-color: $hover-color;
+      transform: translateY(-20px);
+    }
+  }
+}
+
+
+
+
+@for $day_of_mun from 1 through 31 {
+  .calendar__day:nth-child(#{$day_of_mun}):before {
+    content: '#{$day_of_mun}';
+  }
+}
```

Comment:
Try to avoid magic numbers in your code. You can create a variable for `31` to make it clear what this number represents.